### DISCUSSION TITLE
About the dock

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#440] [Ideas]
I think you should know that the Mxc dock is converted from the launcher, so it makes me feel like a filer window, which only shows a list of file shortcuts. 

When the software is opened, the shortcut of the software will be automatically copied there, and it will be automatically removed when it is closed, or if it is pinned, then its removal shortcut command will be removed.

Video
You will understand from 3:45.
https://youtu.be/2GkoAa5718Y

**Comment by probonopd** [comment]
Since I am no longer using a Dock, I will not put additional work into it. But it is still there and anyone can contribute.
